examples/config.json

```diff
@@ -276,14 +276,6 @@
         "gpus"     : 12
     },
 
-    "local.summit" : {
-        "project"  : "BIP178",
-        "queue"    : "batch",
-        "schema"   : "local",
-        "cores"    : 84,
-        "gpus"     : 12
-    },
-
     "ornl.summit_prte" : {
         "project"  : "BIP178",
```

Comment:
This requires a costly work around from the user. E.g., `sed -i 's/BIP178/geo111/g' [...]/lib/python2.7/site-packages/radical/pilot/configs/resource_ornl.json` Do we have a path towards eliminating this hard coded allocation value?

Reply:
Well, this is the example config, not the resource config - so the user should not need to change the resource config file (unless I am missing something).

But FWIW, settings in the resource config can be overwritten by placing a file in `~/.radical/pilot/configs/resource_ornl.json` - any keys defined there would overwrite the installed resource config.